src/NHibernate/Cfg/MappingSchema/Hbm.generated.cs

```diff
@@ -1,6 +1,4 @@
 namespace NHibernate.Cfg.MappingSchema {
-    
-    
```

Comment:
This one is generated. Should we really fix it? I do not fix so, unless we also fix the generator.

Reply:
No, shouldn't